testing/src/test/resources/bootstrap.properties

```diff
@@ -0,0 +1,19 @@
+sql.url=jdbc:mysql://localhost:3306/
+sql.user=root
+sql.password=password
+tenant.id=mtenant
+config.source=UI
+sql.driver=com.mysql.jdbc.Driver
+
+########################################################################
+#AcitveMQ properties
+########################################################################
+jms.queue.for.events=QueueForEvents
+jms.queue.for.scheduler=QueueForScheduler
+jms.broker.url=tcp://localhost:61616
+jms.maximumRedeliveries=0
+jms.redeliveryDelayInMillis=2000
+jms.concurrentConsumers=1
+jms.maxConcurrentConsumers=10
+jms.session.cache.size=10
+jms.cache.producers=false
```

Comment:
I'm not sure how this was working before, but we should ideally be
providing this system level config for our tests to talk to the right
components. Some of our intermittent failures were related to the config
module not finding this file

On Mon, Jun 8, 2015 at 10:14 PM, Frank Huster <notifications@github.com>
wrote:

> In testing/src/test/resources/bootstrap.properties
> <https://github.com/motech-implementations/mim/pull/333#discussion_r31983249>
> :
>
> > +tenant.id=mtenant
> > +config.source=UI
> > +sql.driver=com.mysql.jdbc.Driver
> > +
> > +########################################################################
> > +#AcitveMQ properties
> > +########################################################################
> > +jms.queue.for.events=QueueForEvents
> > +jms.queue.for.scheduler=QueueForScheduler
> > +jms.broker.url=tcp://localhost:61616
> > +jms.maximumRedeliveries=0
> > +jms.redeliveryDelayInMillis=2000
> > +jms.concurrentConsumers=1
> > +jms.maxConcurrentConsumers=10
> > +jms.session.cache.size=10
> > +jms.cache.producers=false
>
> Koshal suggested that this was the cause for the sporadic total failure
> due to the framework not starting.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/333/files#r31983249>.
>